src/bulma.tsx

```diff
@@ -42,9 +42,7 @@ export declare namespace Bulma {
         isLoading?: boolean;
     }
 
-    export type Colors = 'white' | 'light' | 'dark' | 'black' | 'primary' | 'info' | 'success' | 'warning' | 'danger';
     export interface Color {
```

Comment:
It's safe to remove this interface, we're disabling `Colors` _intellisense_ for a greater good.